### DISCUSSION TITLE
MFE in RN?

### DISCUSSION THREAD

**Discussion by domokos-lorant** [module-federation/module-federation-examples#640] [General]
Did anyone give Module federation a try in react native? Is there a sample?

**Comment by jacob-ebey** [comment]
I have not yet explored it. If I remember correctly though, react-native wouldn't support JSONP for chunk loading. Would probably have to figure out a new strategy for loading chunks, but everything else should work. Essentially replacing JsonpChunkLoadingPlugin with something that supports react-native's environment should do it.

**Comment by vitormalencar** [comment]
I would love to see this also, I guess sharing components is a big problem still with RN

**Comment by ScriptedAlchemy** [comment]
Yeah it's been done. I tweeted about it. Also done for electron too

**Reply by priyankagupta23pg** [reply]
> Yeah it's been done. I tweeted about it. Also done for electron too

please share some links for RN ..
Thanks in Advance

**Reply by ScriptedAlchemy** [reply]
Was long ago, google react native repack module federation and something should pop up

**Comment by ScriptedAlchemy** [comment]
Repack supports MF - will likely work on improving it in the future
